Add needed files for _lasx variants. Otherwise we get

/usr/bin/ld: /<<PKGBUILDDIR>>/workdir/GenCxxObject/UnpackedTarball/skia/src/core/SkBitmapProcState_opts.o: in function `SkOpts::Init_BitmapProcState()':
./././workdir/UnpackedTarball/skia/src/core/SkBitmapProcState_opts.cpp:36:(.text+0x58): undefined reference to `SkOpts::Init_BitmapProcState_lasx()'
/usr/bin/ld: /<<PKGBUILDDIR>>/workdir/GenCxxObject/UnpackedTarball/skia/src/core/SkBlitRow_opts.o: in function `SkOpts::Init_BlitRow()':
./././workdir/UnpackedTarball/skia/src/core/SkBlitRow_opts.cpp:36:(.text+0x58): undefined reference to `SkOpts::Init_BlitRow_lasx()'
/usr/bin/ld: /<<PKGBUILDDIR>>/workdir/GenCxxObject/UnpackedTarball/skia/src/core/SkOpts.o: in function `SkOpts::Init()':
./././workdir/UnpackedTarball/skia/src/core/SkOpts.cpp:65:(.text+0x237d8): undefined reference to `SkOpts::Init_lasx()'
/usr/bin/ld: /<<PKGBUILDDIR>>/workdir/GenCxxObject/UnpackedTarball/skia/src/core/SkSwizzler_opts.o: in function `SkOpts::Init_Swizzler()':
./././workdir/UnpackedTarball/skia/src/core/SkSwizzler_opts.cpp:51:(.text+0x58): undefined reference to `SkOpts::Init_Swizzler_lasx()'

diff --git a/external/skia/Library_skia.mk b/external/skia/Library_skia.mk
index 40fe09a29d6a..0cdfe87c68c9 100644
--- a/external/skia/Library_skia.mk
+++ b/external/skia/Library_skia.mk
@@ -1309,6 +1309,15 @@ $(eval $(call gb_Library_add_generated_exception_objects,skia,\
 	$(LO_SKIA_AVOID_INLINE_COPIES) \
 ))
 
+ifeq ($(CPUNAME),LOONGARCH64)
+$(eval $(call gb_Library_add_generated_exception_objects,skia,\
+    UnpackedTarball/skia/src/opts/SkOpts_lasx \
+    UnpackedTarball/skia/src/core/SkSwizzler_opts_lasx \
+    UnpackedTarball/skia/src/core/SkBlitRow_opts_lasx \
+    UnpackedTarball/skia/src/core/SkBitmapProcState_opts_lasx, \
+        $(LO_SKIA_AVOID_INLINE_COPIES) \
+))
+endif
 
 # Skcms code is used by png writer, which is used by SkiaHelper::dump(). Building
 # this without optimizations would mean having each pixel of saved images be
